### PR TITLE
Add doctest to the build

### PR DESCRIPTION
**Pull request by jystic**

### CODE REVIEW

File: hedgehog/src/Hedgehog.hs [hunk 1]
```diff
@@ -37,9 +37,9 @@
 --
 -- You can then load the module in GHCi, and run it:
 --
--- >>> tests
--- ━━━ Test.Example ━━━
---   ✓ prop_reverse passed 100 tests.
+-- > λ tests
```
Comment (moodmosaic) [inline hedgehog/src/Hedgehog.hs]: 👍 for the fancy λάμδα

Comment (jystic) [inline hedgehog/src/Hedgehog.hs]: Add it your `~/.ghc/ghci.conf` for extra points 🚀 
```
:set prompt "\ESC[34m\STX%s\n\ESC[0;33m\STXλ \ESC[m\STX"
```
screen shot 2017-04-12 at 1 23 38 pm

Comment (moodmosaic) [inline hedgehog/src/Hedgehog.hs]: It's there already + on my keyboard as well 😉 

c3-ufqzxuaaymmi

Comment (jystic) [inline hedgehog/src/Hedgehog.hs]: Nice 👍 fancy

**Pull request merged**